diff --git a/index.js b/index.js
index 8c5a44edb4b5515a33e45bc4d1caa15c1936d211..deae96bb79921cbc7bb950462278810f53224e0f 100644
--- a/index.js
+++ b/index.js
@@ -7,5 +7,5 @@ const {promisify} = require('util')
 /* Shallow clone so we can promisify in-place */
 const fs = { ...require('fs') }
 const {spawn} = require('cross-spawn')
-const rimraf = promisify(require('rimraf'))
+const { rimraf } = require('rimraf')
 const pMap = require('p-map')
