diff --git a/index.js b/index.js
index 3164b1d0b40f7873aadb91798471a35f0d422405..26d82a6f2e3adcae577f5426a328a8a5f4fff77f 100755
--- a/index.js
+++ b/index.js
@@ -16,7 +16,7 @@ const mkdirp = require('make-dir')
 const Module = require('module')
 const onExit = require('signal-exit')
 const path = require('path')
-const rimraf = promisify(require('rimraf'))
+const { rimraf } = require('rimraf')
 const SourceMaps = require('./lib/source-maps')
 const TestExclude = require('test-exclude')
 const pMap = require('p-map')
diff --git a/lib/commands/instrument.js b/lib/commands/instrument.js
index cabb437759f06aa671a733399b27ac57b63c3a3f..23500b04800197ffc4ddb9308f17a1500c71a09b 100644
--- a/lib/commands/instrument.js
+++ b/lib/commands/instrument.js
@@ -4,5 +4,5 @@ const NYC = require('../../index.js')
 const path = require('path')
 const { promisify } = require('util')
 const resolveFrom = require('resolve-from')
-const rimraf = promisify(require('rimraf'))
+const { rimraf } = require('rimraf')
 const { cliWrapper, setupOptions } = require('./helpers.js')
